GeoEntity.md

```diff
@@ -17,10 +17,10 @@
 ----------------------------
 #Description
 
-This service provides aggregated geography entities. Currently the following requests could be made.
+This service provides lists of geographic entities of a particular type, such as postal codes or first-level administrative areas. This service uses the same underlying data as our geocoding service does. The list of entities returned as quried should be comprehensive for the populous countries, but we could not guarantee the completeness. We are considering to provide features to add missing entities if there are any and as customers request. Currently, the following requests are supported:
```

Comment:
"as our geocoding service does" --> either "as our geocoding service" or "that our geocoding service does"

typo in the next sentence: "quried"

"but we could not guarantee the completeness" --> "but we cannot guarantee 100% completeness" ("could" implies conditionality here, as in "we could not guarantee" [...if so-and-so were true])

"We are considering... as customers request" --> I would rephrase this sentence and move it immediately after the list of requests, before the GET/POST line (and don't forget, please kill the POST support on this service). maybe something like "If you require lists of a geographic entity type that is not on this list, please contact [team email]"